packages/utils/interactive-hooks/src/index.ts

```diff
@@ -61,3 +61,4 @@ export { preferKeyDownForKeyEvents, useKeyCallback, useKeyDownProps, useKeyProps
 export type { KeyCallback, KeyPressProps } from './useKeyProps';
 export { useOnPressWithFocus } from './useOnPressWithFocus';
 export type { OnPressCallback, OnPressWithFocusCallback } from './useOnPressWithFocus';
+export { getAccessibilityState, getAccessibilityStateWorker } from './getAccessibilityState';
```

Comment:
You shouldn't need to export the worker, getAccessibilityState should be all that needs to be referenced.

Reply:
Unless you're exporting it for client extensibility?